cmd/apiserver/main.go

```diff
@@ -234,7 +247,24 @@ func main() {
 
 				store := inmem.New()
 
-				api, err := handlers.NewAPI(ctx, logger.Sugar(), db, ipam, fflags, store, signalBus)
+				sessionStore := cookie.NewCookieStore(
```

Comment:
Style nit - would prefer to set it like this.
But do we still need to support both?
```go
var sessionStore session.Store
if redisServer != "" {
  // setup redis store
} else {
  // setup cookie store
}
```

Reply:
You're likely right.. we only test the redis case anyways.  I'll comment it out just in case we want to bring it back in the future.